accessing
noteError: anError
	self halt.